src/storage/src/internal_control.rs

```diff
@@ -19,6 +19,17 @@ use mz_storage_client::controller::CollectionMetadata;
 use mz_storage_client::types::sinks::{MetadataFilled, StorageSinkDesc};
 use mz_storage_client::types::sources::IngestionDescription;
 
+/// Storage instance configuration parameters that can affect
+/// dataflow rendering, and as such must be applied to
+/// `StorageWorker`s in a consistent order with source and sink
+/// creation.
+#[derive(Clone, Debug, Default, Serialize, Deserialize, PartialEq, Eq)]
+pub struct DataflowParameters {
+    /// Controls whether or not to use the new storage `persist_sink` implementation in storage
+    /// ingestions.
+    pub enable_new_storage_persist_sink: bool,
```

Comment:
Can we find a better name? `enable_multi_worker_persist_sink`?

Reply:
yes!

Reply:
Fixed in https://github.com/MaterializeInc/materialize/pull/18010